dashboard/config/database.yml

```
@@ -73,6 +78,11 @@ environment_defaults: &environment_defaults
   primary_reporting:
     replica: true
     <<: *mysql_defaults
+    <% if !CDO.running_web_application? %>
```

tiny nit: [Prefer `unless` over `if` for negative conditions](https://github.com/rubocop/ruby-style-guide#if-vs-unless)

```suggestion
    <% unless CDO.running_web_application? %>
```